python/paddle/fluid/tests/unittests/test_inplace.py

```diff
@@ -0,0 +1,94 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import numpy as np
+
+import paddle
+import paddle.fluid.core as core
+
+
+class TestInplace(unittest.TestCase):
+    def test_forward_version(self):
+        with paddle.fluid.dygraph.guard():
```

Comment:
Should we delete these `with paddle.fluid.dygraph.guard()` since now dygraph is enabled by default

Reply:
In `TestInplace`, different test_XX functions interact with each other and have the same `Tracer`, which will cause  the tests to fail. So call  `with paddle.fluid.dygraph.guard()` to use different `Tracer` in different test.